simple_classifier/Splitter/Splitter_subclasses.py

```diff
@@ -0,0 +1,42 @@
+from simple_classifier.Splitter.Splitter import Splitter
+from sklearn.model_selection import train_test_split
+
+
+@Splitter.register("Percentage_split")
```

Comment:
Let's keep codenames all in lower case for consistency -> `percentage_split`. What do you think?